src/reactions/MessageReactionList.js

```diff
@@ -86,7 +86,7 @@ const getReactionsTabs = (
       // that's since been removed.  Ignore the nav hint in that case.
       ...(reactionName !== undefined && reactionsTabs[reactionName]
         ? { initialRouteName: reactionName }
-        : {}),
+        : { initialRouteName: undefined }),
```

Comment:
Perhaps `Object.freeze({})` here too?

Otherwise, this code would be simplified by inlining the spread and just having an `initialRouteName` property with a conditional expression inside its initializer.